src/spyglass/utils/dj_mixin.py

```diff
@@ -493,3 +493,14 @@ def super_delete(self, *args, **kwargs):
         logger.warning("!! Using super_delete. Bypassing cautious_delete !!")
         self._log_use(start=time(), super_delete=True)
         super().delete(*args, **kwargs)
+
+
+class SpyglassGroupPart(SpyglassMixin, dj.Part):
+    """
+    A part table for Spyglass Group tables. Assists in propagating
+    delete calls from upstreeam tables to downstream tables.
+    """
+
+    def delete(self, key={}):
```

Comment:
- I'd avoid default `{}` or `[]` as a lesson I learned from [`dataclass` factories](https://docs.python.org/3/library/dataclasses.html#dataclasses.field). Default values are mutable so using them as defaults might lead to a case where some part of a function edits the default value, which would have unexpected consequences on subsequent runs. Not the case here, but good habit.
- For tables, I find `table & True` easier to read than `table & {}`. The default could be `True`, but that might confuse someone into thinking the default type were `bool`, so I prefer `None`

```suggestion
    def delete(self, key: dict = None):
        key = key or True
```